igtdetect/pdf2gloss.py

```diff
@@ -71,11 +76,17 @@ def scan_pdfs(input_path, temp_path):
                 scanned_count += 1
             except:
                 logging.error('PDF scan failed for: {}'.format(filename))
+
+            # get the doi from the pdf and save it into the dois dict
+            identifier_result = pdf2doi.pdf2doi(str(path_to_pdf))
+            dois[os.path.splitext(filename)[0]] = identifier_result['identifier']
+            if identifier_result['identifier'] is None:
+                logging.error('pdf2doi was not able to find a doi for {}'.format(filename))
```

Comment:
You're still returning `None` for that document. Maybe move line 82 into an `else` clause